adapters/repos/db/vector/hnsw/index_test.go

```diff
@@ -37,47 +37,47 @@ func TestHnswIndex(t *testing.T) {
 	require.Nil(t, err)
 
```

Comment:
whole file: see above, all node ids should be represented by `uint64`, removing the need for the conversions.